packages/rancher-tracing/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: local
-packageVersion: 01
+packageVersion: 02
```

Comment:
is this one supposed to be RC version increase or did we do package version for a reason here?

Reply:
I tried to bump the RC version on this one and it failed validation `new-assets/charts-without-rc/rancher-tracing/rancher-tracing/1.20.001 does not match that of old contents at original-assets/charts/rancher-tracing/rancher-tracing/1.20.001`. I bumped the package version since the error message references "without-rc".

Reply:
1.20.001 was already released: https://github.com/rancher/charts/tree/main/assets/rancher-tracing
So this is working as intended.
@aiyengar2 you had listed this as new version, is there problem there ? Hoping it got generated correctly.

Reply:
gotcha okay then that should be fine!

Reply:
Sounds like there were no changes. So we will just bump this to 1.20.002, I added that to the "new charts" issue.

Reply:
Yes there was no need for a change for rancher-tracing since it was a local chart with no CRD chart annotation (so the only change was to the rcVersion and make sync did not fail in the initial migration of the charts).

LGTM